src/main/java/chat/Parser.java

```diff
@@ -0,0 +1,38 @@
+package chat;
+
+import chat.command.*;
+
+/**
+ * Parser class deals with making sense of user command.
+ */
+
+public class Parser {
+
+    /**
+     * Parses command to their respective commands.
+     * 
+     * @param str Inputted command string from user to Chat the Cat.
+     * @return Command object.
+     * @throws ChatException If command is not known by Chat the Cat.
+     */
+    public static Command parse(String str) throws ChatException {
+        if (str.equals("list")) {
+            return new ListCommand();
+        } else if (str.startsWith("done")) {
+            return new DoneCommand(str);
+        } else if (str.startsWith("todo") | str.startsWith("deadline") | str.startsWith("event")) {
+            return new AddCommand(str);
+        } else if (str.startsWith("delete")) {
+            return new DeleteCommand(str);
+        } else if (str.equals("bye")) {
+            return new ExitCommand();
+        } else if (str.startsWith("find")) {
+            return new FindCommand(str);
+        } else {
```

Comment:
Perhaps using a switch statement will be clear and easier to maintain?

Reply:
Yep I think that will be better, thanks for the suggestion 😄.